defaults/main.yml

```diff
@@ -43,8 +43,8 @@ apache_options:
   - "-Indexes"
   - "+FollowSymLinks"
 
-apache_ssl_protocol: "All -SSLv2 -SSLv3"
-apache_ssl_cipher_suite: "AES256+EECDH:AES256+EDH"
+apache_ssl_protocol: "All -SSLv2 -SSLv3 -TLSv1 -TLSv1.1"
+apache_ssl_cipher_suite: "ECDHE-ECDSA-AES128-GCM-SHA256:ECDHE-RSA-AES128-GCM-SHA256:ECDHE-ECDSA-AES256-GCM-SHA384:ECDHE-RSA-AES256-GCM-SHA384:ECDHE-ECDSA-CHACHA20-POLY1305:ECDHE-RSA-CHACHA20-POLY1305:DHE-RSA-AES128-GCM-SHA256:DHE-RSA-AES256-GCM-SHA384"
```

Comment:
For better readability `apache_ssl_cipher_suite` should be a list. But to be backward compatible this would require a some checks e.g. `{{ apache_ssl_cipher_suite | join(':') if apache_ssl_cipher_suite is iterable and apache_ssl_cipher_suite is not string else apache_ssl_cipher_suite }}`. Otherwise it would be a breaking change.